storage/local/storage.go

```diff
@@ -151,14 +151,13 @@ func NewMemorySeriesStorage(o *MemorySeriesStorageOptions) (Storage, error) {
 			Name:      "persist_latency_microseconds",
 			Help:      "A summary of latencies for persisting each chunk.",
 		}),
-		persistErrors: prometheus.NewCounterVec(
+		persistErrors: prometheus.NewCounter(
```

Comment:
Nitpick: Now that you have only one arg to this function, you can put it on one line (as done elsewhere):

    persistErrors: prometheus.NewCounter(prometheus.CounterOpts{
		 				Namespace: namespace, // ...